src/registrar/tests/test_views_domain.py

```diff
@@ -424,6 +424,112 @@ def test_domain_readonly_on_detail_page_for_org_admin_not_manager(self):
         self.assertContains(detail_page, "invited@example.com")
 
 
+class TestDomainDetailDomainRenewal(TestDomainOverview):
+    def setUp(self):
+        super().setUp()
+
+        self.user = get_user_model().objects.create(
+            first_name="User",
+            last_name="Test",
+            email="bogus@example.gov",
+            phone="8003111234",
+            title="test title",
+            username="usertest",
+        )
+
+        self.expiringdomain, _ = Domain.objects.get_or_create(
+            name="expiringdomain.gov",
+        )
+
+        UserDomainRole.objects.get_or_create(
+            user=self.user, domain=self.expiringdomain, role=UserDomainRole.Roles.MANAGER
+        )
+
+        DomainInformation.objects.get_or_create(creator=self.user, domain=self.expiringdomain)
+
+        self.portfolio, _ = Portfolio.objects.get_or_create(organization_name="Test org", creator=self.user)
+
+        self.user.save()
+
+    def custom_is_expired(self):
+        return False
+
+    def custom_is_expiring(self):
+        return True
+
+    @override_flag("domain_renewal", active=True)
+    def test_expiring_domain_on_detail_page_as_domain_manager(self):
```

Comment:
suggest simplifying this test name eg test_expiring_domain_domain_manager_view. same nit suggestion for the 2 test names below

Reply:
Open to negotiation (😂) on editing the name, but I wanted to leave `detail_page` in just so this test specifically recognizes which part of the "form" it's for (ie the domain part on the detail page)